app/src/main/java/com/example/checkpoint/dao/IWeather.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.checkpoint.dao
+
+import com.example.checkpoint.dto.WeatherAPI
+import retrofit2.Call
+import retrofit2.http.GET
+
+
+
+interface IWeather {
+    //TODO("need to change lat and ln to use the andriod info.")
+    //NOTE: The current is a example
+    //var lon : String
+    //var lat : String
+    //fun location(lon : String, lat : String ){
+      //  this.lon = lon
+       // this.lat = lat
+    //}
+    //something is not working
+    //var websiteLink: String
+        //get() = "/data/2.5/weather?lat=$lat&lon=$lon&appid=707f43b75fdd2f4d0ae87ccf8b6764f3"
+        //set(value) = TODO("I wish I knew")
+
+    //@GET(websiteLink)
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.